deployments/mumbai/usdc/migrations/1676659582_configurate_and_ens.ts

```diff
@@ -0,0 +1,255 @@
+      //   signature: 'depositFor(address,address,bytes)',
+      //   calldata: depositForCOMPCCalldata
+      // },
+
+      // Note: No need to do this because we have already set up the subdomain on Goerli
+      // 6. Set up ENS license subdomain with the Timelock as the owner
+      // {
+      //   target: ENSRegistryAddress,
+      //   signature: 'setSubnodeRecord(bytes32,bytes32,address,address,uint64)',
+      //   calldata: ethers.utils.defaultAbiCoder.encode(
+      //     ['bytes32', 'bytes32', 'address', 'address', 'uint64'],
+      //     [nameHash, labelHash, timelock.address, ENSResolverAddress, ttl]
+      //   )
+      // },
+
+      // 7. Establish the new list of official markets
+      {
+        target: ENSResolverAddress,
+        signature: 'setText(bytes32,string,string)',
+        calldata: ethers.utils.defaultAbiCoder.encode(
+          ['bytes32', 'string', 'string'],
+          [subdomainHash, ENSTextRecordKey, JSON.stringify(officialMarkets)]
+        )
+      },
+    ];
+
+    const description = "Configurate Mumbai cUSDCv3 market, bridge over USDC and COMP, and update ENS text record.";
+    const txn = await govDeploymentManager.retry(async () =>
+      trace(await governor.propose(...(await proposal(mainnetActions, description))))
+    );
+
+    const event = txn.events.find(event => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+
+    trace(`Created proposal ${proposalId}.`);
+  },
+
+  async verify(deploymentManager: DeploymentManager, govDeploymentManager: DeploymentManager) {
+    const ethers = deploymentManager.hre.ethers;
+
+    const {
+      comet,
+      rewards,
+      DAI,
+      WBTC,
+      WETH,
+      WMATIC
+    } = await deploymentManager.getContracts();
+
+    const {
+      timelock,
+    } = await govDeploymentManager.getContracts();
+
+    // 1.
+    const daiInfo = await comet.getAssetInfoByAddress(DAI.address);
+    const wbtcInfo = await comet.getAssetInfoByAddress(WBTC.address);
+    const wethInfo = await comet.getAssetInfoByAddress(WETH.address);
+    const wmaticInfo = await comet.getAssetInfoByAddress(WMATIC.address);
+    expect(await daiInfo.supplyCap).to.be.eq(exp(1_000_000, 18));
+    expect(await wbtcInfo.supplyCap).to.be.eq(exp(20_000, 8));
+    expect(await wethInfo.supplyCap).to.be.eq(exp(50_000, 18));
+    expect(await wmaticInfo.supplyCap).to.be.eq(exp(500_000, 18));
+
+    // 2. & 3.
+    // Note: Cannot verify because the USDC we are bridging over is different from the one in Comet
+    // expect(await comet.getReserves()).to.be.equal(exp(400_000, 6));
+    const bridgedUSDC = new Contract(
+      '0x0FA8781a83E46826621b3BC094Ea2A0212e71B23',
+      ERC20__factory.createInterface(),
+      deploymentManager.hre.ethers.provider
+    );
+    expect(await bridgedUSDC.balanceOf(comet.address)).to.be.equal(exp(500, 6));
+
+    // 4. & 5.
+    // const bridgedCOMP = new Contract(
+    //   '0x46DaBF9092B40A3fE6c0bC3331Cd928B600754fE',
```

Comment:
What is this address?

Reply:
This is the Mumbai COMP that I mapped using the [Polygon mapper](https://mapper.polygon.technology/). Unfortunately, we can't bridge it using the `RootChainManager` approach.